### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by policetonyR** [jwoglom/tconnectsync#31]
I have tried the latest version. I believe the CSV format was updated or something. I tried printing out the sections but all I get is an empty array for each.

![image](https://user-images.githubusercontent.com/31581859/152663934-1c9a5430-6863-4149-887e-1fe6640b5a1f.png)

**Comment by policetonyR** [comment]
Seems to be working now? I do not know what went wrong before.

**Comment by jwoglom** [comment]
Just wondering, at what point did you set up tconnectsync relative to setting up your pump? This might have occurred because there was no data in tconnect. I've also been trying to track down whether it's required or not to upload pump data via your PC once before using the android app, for tconnectsync to be able to find the data properly. So any insight into the exact situation you were in 5 days ago relative to now would be helpful.

**Comment by policetonyR** [comment]
No, I have done this before. I believe it may have stopped uploading data because I had to log back into tconnect on my iPhone. That is the only thing I can think of, otherwise I have no clue.

**Comment by jwoglom** [comment]
The app will stop uploading data once it logs out, unfortunately. On Android, I typically try to open the app at least once per day to make sure it's still uploading. I could investigate adding a feature in tconnectsync to alert/send an email or something when there haven't been readings for >12 hours to help make it more obvious when this happens.

Glad you got it working!

**Comment by policetonyR** [comment]
Yea I have my tconnect app on my iPhone always open. I try to run the python file at least once a day or so. I am just confused on what happened. I believe it may have been the logged out issue, but I do not exactly know. Ty!
